src/Ladybug_SolarEnvelopeBasic.py

```diff
@@ -180,11 +180,11 @@ def __calc(self):
   
         eqtime = 4*math.degrees(vary*math.sin(2*math.radians(Mlong))-2*Eccent*math.sin(math.radians(Manom))+4*Eccent*vary*math.sin(math.radians(Manom))*math.cos(2*math.radians(Mlong))-0.5*vary*vary*math.sin(4*math.radians(Mlong))-1.25*Eccent*Eccent*math.sin(2*math.radians(Manom)))
 
-        hourangle= math.degrees(math.acos(math.cos(math.radians(90.833))/(math.cos(math.radians(latitude))*math.cos(math.radians(declination)))-math.tan(math.radians(latitude))*math.tan(math.radians(declination))))
+        #hourangle= math.degrees(math.acos(math.cos(math.radians(90.833))/(math.cos(math.radians(latitude))*math.cos(math.radians(declination)))-math.tan(math.radians(latitude))*math.tan(math.radians(declination))))
 
         self.solarnoon_t=(720-4*longitude-eqtime+timezone*60)/1440
-        self.sunrise_t  =self.solarnoon_t-hourangle*4/1440
-        self.sunset_t   =self.solarnoon_t+hourangle*4/1440
+        #self.sunrise_t  =self.solarnoon_t-hourangle*4/1440
```

Comment:
@saeranv is this correct and you really want to comment these lines from the code? I can merge this in once you confirm.